include/wx/cmdargs.h

```diff
@@ -31,19 +31,17 @@ class WXDLLIMPEXP_BASE wxCmdLineArgsArray
     wxCmdLineArgsArray() { m_argsA = NULL; m_argsW = NULL; }
 
     template <typename T>
-    wxCmdLineArgsArray& operator=(T **argv)
+    void Init(int argc, T **argv)
     {
         FreeArgs();
 
         m_args.clear();
+        m_args.reserve(argc);
 
-        if ( argv )
+        for ( int i = 0; i < argc; i++ )
         {
-            while ( *argv )
-                m_args.push_back(*argv++);
+            m_args.push_back(*argv++);
```

Comment:
This line isn't new but since it's somewhat changed anyway: wouldn't `argv[i]` be slightly better form? As it never lets argv point out of bounds (yes, likely just one out of a gazillion things a memory debugger would trip on).